spec/features/feedback_spec.rb

```diff
@@ -80,6 +80,8 @@
       end
 
       background do
+        application.authority.update_attribute(:write_to_councillors_enabled, true)
```

Comment:
http://api.rubyonrails.org/v3.2.22.1/classes/ActiveRecord/Persistence.html#method-i-update_attribute

Validations are skipped with `#update_attribute`. I think I might have been defaulting to using this in tests so that they're isolated from validations and whatever else we have/add.

All we care about in the test when we toggle this attribute is that the value is true/false, not whether the instance still validates after the change. Using `update!` does make the test a little less focused then. We just chatted about this, and thought that we wouldn't want the use of `update_attribute` to slip into the app code, so better/fine to use `update!` anyway.